cmd/zetaclientd/init.go

```diff
@@ -106,8 +120,36 @@ func Initialize(_ *cobra.Command, _ []string) error {
 	configData.KeyringBackend = config.KeyringBackend(initArgs.KeyringBackend)
 	configData.HsmMode = initArgs.HsmMode
 	configData.HsmHotKey = initArgs.HsmHotKey
+	configData.SolanaKeyFile = initArgs.SolanaKey
 	configData.ComplianceConfig = testutils.ComplianceConfigTest()
 
-	//Save config file
+	// Save solana fee payer key
+	keyFile := path.Join(rootArgs.zetaCoreHome, initArgs.SolanaKey)
+	err = createSolanaKeyFile(keyFile)
+	if err != nil {
+		return err
+	}
+
+	// Save config file
 	return config.Save(&configData, rootArgs.zetaCoreHome)
 }
+
+// createSolanaKeyFile creates a solana key json file
+func createSolanaKeyFile(keyFile string) error {
```

Comment:
Can we move this and `solanaTestKey` to a separate file like `solana_test_key` to make clear separation what is experimental

Reply:
https://github.com/zeta-chain/node/pull/2560/commits/ada365437e0782d6d2f6725ca87d6c53a687118d